unparser.gemspec

```diff
@@ -19,7 +19,7 @@ Gem::Specification.new do |gem|
 
   gem.required_ruby_version = '>= 1.9.3'
 
-  gem.add_dependency('parser',        '~> 2.2.0.2')
+  gem.add_dependency('parser',        '~> 2.2', '>= 2.2.0.2')
```

Comment:
I think that we should depend on `~> 2.2`, to keep this dependency simple. I do not know anymore why I choose `~> 2.2.0.2`, thats overly specific.